app/scripts/lib/rpc-method-middleware/handlers/add-ethereum-chain.test.js

```diff
@@ -541,4 +542,42 @@ describe('addEthereumChainHandler', () => {
       }),
     );
   });
+
+  it('should add result set to null to response object if the requested rpcUrl (and chainId) is currently selected', async () => {
+    jest.spyOn(chainUtils, 'findExistingNetwork').mockReturnValue({
+      chainId: '0x1',
+      rpcUrl: 'https://mainnet.infura.io/v3/',
+      ticker: 'ETH',
+    });
```

Comment:
guessing you can just modify the `findNetworkConfigurationBy` mock result because `findExistingNetwork` also checks the built in networks constant?

if so, can we modify this test to use a non-built-in network instead?

Reply:
if it was clear, this is a nit and totally ignorable. I'm mainly asking so we can avoid having to import and mock chainutils

Reply:
addressed here: https://github.com/MetaMask/metamask-extension/pull/26726/commits/4c9364078ea8975b1a02189a36559b78eceaf23e